bin/package-requires-update

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env node
```

Comment:
Fixed in: https://github.com/belcherj/wp-calypso/commit/18284114c931b4e19f15643b2d39843da216b362